### PR TITLE
Add 'file.output.formats'

### PR DESCRIPTION
**Pull request by 0ncorhynchus**

### CODE REVIEW

File: src/afmize.cpp [hunk 1]
```diff
@@ -396,18 +454,42 @@ int main(int argc, char** argv)
...
+            }
+
+            if(output_formats.json)
+            {
+                afmize::write_json(stg, outname);
+            }
+
+            if(output_formats.ppm)
+            {
+            }
```
Comment (ToruNiina) [inline src/afmize.cpp]: It seems that you forgot to output ppm, no?

**Review by ToruNiina — CHANGES_REQUESTED** [review]
Thank you for the new feature! This is clearly helpful for many users, so basically I'm positive to merge it.

But I found one small thing that may be just a tiny mistake. So I'm still not fully convinced about merging it. Would you check my comment?

File: src/afmize.cpp (hunk 1 above)
Comment (0ncorhynchus) [reply src/afmize.cpp]: I forgot to remove these lines. I ensured to output ppm with the below lines.
I'll remove these.

Comment (ToruNiina) [reply src/afmize.cpp]: 👌

**Review by ToruNiina — APPROVED** [review]

**Pull request merged**